src/assets/stylesheets/create-object-dialog.scss

```diff
@@ -0,0 +1,74 @@
+@import 'shared';
+
+:local(.addMediaForm) {
```

Comment:
i'm pretty sure we've been naming these styles in `snake-case`, and then the react/webpack stuff will case them in JS land as you have here.